seeds.sql

```diff
@@ -1,11 +1,2532 @@
+(44, 11, 'https://idp10.com/book44', 'Access Book 44'),
+(29, 2, 'https://idp19.com/book29', 'Access Book 29'),
+(5, 18, 'https://idp12.com/book5', 'Access Book 5'),
+(45, 8, 'https://idp3.com/book45', 'Access Book 45'),
+(34, 5, 'https://idp18.com/book34', 'Access Book 34'),
+(1, 1, 'https://idp18.com/book1', 'Access Book 1'),
+(27, 17, 'https://idp7.com/book27', 'Access Book 27'),
+(8, 7, 'https://idp16.com/book8', 'Access Book 8'),
+(4, 1, 'https://idp18.com/book4', 'Access Book 4'),
+(47, 2, 'https://idp1.com/book47', 'Access Book 47'),
+(49, 6, 'https://idp14.com/book49', 'Access Book 49'),
+(50, 18, 'https://idp16.com/book50', 'Access Book 50'),
+(18, 1, 'https://idp6.com/book18', 'Access Book 18'),
+(44, 7, 'https://idp13.com/book44', 'Access Book 44'),
+(37, 4, 'https://idp5.com/book37', 'Access Book 37'),
+(11, 2, 'https://idp19.com/book11', 'Access Book 11'),
+(42, 7, 'https://idp13.com/book42', 'Access Book 42'),
+(38, 16, 'https://idp18.com/book38', 'Access Book 38'),
+(12, 20, 'https://idp13.com/book12', 'Access Book 12'),
+(29, 18, 'https://idp6.com/book29', 'Access Book 29'),
+(26, 12, 'https://idp14.com/book26', 'Access Book 26'),
+(13, 2, 'https://idp14.com/book13', 'Access Book 13'),
+(45, 16, 'https://idp3.com/book45', 'Access Book 45'),
+(50, 17, 'https://idp6.com/book50', 'Access Book 50'),
+(36, 14, 'https://idp1.com/book36', 'Access Book 36'),
+(31, 10, 'https://idp20.com/book31', 'Access Book 31'),
+(21, 14, 'https://idp15.com/book21', 'Access Book 21'),
+(39, 17, 'https://idp3.com/book39', 'Access Book 39'),
+(32, 5, 'https://idp6.com/book32', 'Access Book 32'),
+(27, 7, 'https://idp20.com/book27', 'Access Book 27'),
+(8, 1, 'https://idp11.com/book8', 'Access Book 8'),
+(7, 9, 'https://idp6.com/book7', 'Access Book 7'),
+(22, 7, 'https://idp19.com/book22', 'Access Book 22'),
+(47, 11, 'https://idp3.com/book47', 'Access Book 47'),
+(19, 3, 'https://idp20.com/book19', 'Access Book 19'),
+(30, 2, 'https://idp8.com/book30', 'Access Book 30'),
+(21, 13, 'https://idp12.com/book21', 'Access Book 21'),
+(5, 6, 'https://idp13.com/book5', 'Access Book 5'),
+(34, 18, 'https://idp9.com/book34', 'Access Book 34'),
+(17, 2, 'https://idp8.com/book17', 'Access Book 17'),
+(26, 16, 'https://idp12.com/book26', 'Access Book 26'),
+(16, 6, 'https://idp6.com/book16', 'Access Book 16'),
+(27, 2, 'https://idp20.com/book27', 'Access Book 27'),
+(32, 12, 'https://idp13.com/book32', 'Access Book 32'),
+(23, 11, 'https://idp13.com/book23', 'Access Book 23'),
+(6, 18, 'https://idp10.com/book6', 'Access Book 6'),
+(43, 8, 'https://idp8.com/book43', 'Access Book 43'),
+(10, 7, 'https://idp20.com/book10', 'Access Book 10'),
+(48, 16, 'https://idp1.com/book48', 'Access Book 48'),
+(22, 13, 'https://idp10.com/book22', 'Access Book 22'),
+(46, 4, 'https://idp13.com/book46', 'Access Book 46');
+
+INSERT INTO `highlight` (`id`, `user_id`, `book_id`, `spineIdRef`, `cfi`, `color`, `note`, `updated_at`, `deleted_at`) VALUES
```

Comment:
This data is specific to epub provided. If you are going to use non-copyrighted books to extract the corresponding values like `spineIdRef` and `cfi` then please do so, but otherwise leave blank.

For almost all of this data, a better set of test data might be to start up the app, ingest non-copyrighted books, perform various actions like highlighting and then dump the database for future tests.